### DISCUSSION TITLE
Recommendation Tab

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [community/community#12605] [Other Feature Feedback, Questions, & Ideas]
There should be a way to be like "I don't like this suggestion", in the "Explore Repo" section on the main page of github like youtube has for the recommendation tab, if it doesn't already exist.

Please look at 2 stated by @TechStudent11.
![image](https://user-images.githubusercontent.com/84712013/157485131-72957171-e2d9-4461-9f09-ccd981a83c7f.png)

**Comment by TechStudent10** [comment]
Dup of #10297

**Reply by KendallDoesCoding** [reply]
> Dup of #10297

I don't think so coz in that discussion you didn't put a solution to the issue, I've put the suggestion as

> There should be a way to be like "I don't like this suggestion", in the "Explore Repo" section on the main page of github like youtube has for the recommendation tab, if it doesn't already exist.
